.github/workflows/unit-test.yml

```diff
@@ -26,3 +26,6 @@ jobs:
 
       - name: Test
         run: make test
+
+      - name: Integration Test
+        run: make integration_test
```

Comment:
`make test` is covered `./tests`, so we don't need to introduce extra `make integration_test` target.